src/epcpm/cli/sunspectostaticmodbus.py

```diff
@@ -28,37 +28,57 @@ def map_type_uuid(input, sunspec_types, staticmodbus_types):
     raise ValueError("Error: failure to map type")
 
 
-def create_data_object(
-    input_sunspec_data, scale_factor_uuid_map, sunspec_types, staticmodbus_types
+def create_data_objects(
+    input_sunspec_csv, scale_factor_uuid_map, sunspec_types, staticmodbus_types
```

Comment:
type hints por favor.

Reply:
Added type hints in 73afd44b3cf2d463467b95ab4664543c5ee732e7